Extension/package-lock.json

```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-tE1SYtNG3I3atRVPELSGN2FJJJtPg3O/G0tycYSyzeDqdAbdLPRH089LhpWYA5M/iHeWHkVZq/b0OVKngcK0Eg==",
       "dev": true,
       "requires": {
-        "@types/node": "8.5.4"
+        "@types/node": "*"
```

Comment:
this is strange.  I wonder why this was not updated by the merge from master.

Reply:
I think master is incorrect.

Reply:
no, master is correct. this is wrong.  I'll see if I can fix it